Source/SwiftLintFramework/Rules/ConditionalBindingCascadeRule.swift

```diff
@@ -36,12 +37,10 @@ public struct ConditionalBindingCascadeRule: Rule {
     )
 
     public func validateFile(file: File) -> [StyleViolation] {
-        let conditionalBindingKeywords = ["if", "guard"]
-        let pattern =  "^(" +
-                        conditionalBindingKeywords.joinWithSeparator("|") +
-                        ")(\\s*?)let((.|\\s)*?),(\\s*?)let((.|\\s)*?)\\{"
-        return file.matchPattern(pattern,
-            excludingSyntaxKinds: SyntaxKind.commentAndStringKinds()).map {
+        return file.matchPattern("^(if|guard)(.*?)let(.*?),(.*?)let(.*?)\\{",
+            excludingSyntaxKinds: SyntaxKind.commentAndStringKinds()).filter {
+                !(file.contents as NSString).substringWithRange($0).containsString("where")
```

Comment:
I'm sure this could lead to some missed violations, but I'm not too concerned with that right now.